app/main.development.js

```diff
@@ -195,7 +195,21 @@ ipcMain.on("get-dcrwallet-logs", (event) => {
 });
 
 ipcMain.on("get-decrediton-logs", (event) => {
-  event.returnValue = "decrediton logs!";
+  const logFileName = logger.transports.file.dirname + "/" +logger.transports.file.filename;
+  fs.readFile(logFileName, (err, data) => {
```

Comment:
Yeah, I don't really like this approach. This is reading the entire decrediton log file into memory every time this ipc endpoint is requested (ie, every 2 seconds once the user accesses the log page).

An improvement over this would be to use `fs.Open` and a function to copy data backwards from the end of the file (I don't know if there's anything ready for that, so you'd need to do a bit of research or implement it manually using the fs module).

Another option would be to hook the `main-log` IPC receiver, and add to a memory buffer that will hold the last log lines (like what is done for dcrd/dcrwallet).

Reply:
fixed. This PR is ready for another review